integration/int_3Ri_2BhaRi2_3Re_2BhaRe3/deploy_ssh.yml

```diff
@@ -15,18 +15,38 @@
       with_items: [Router_I1, Router_I2, Router_I3, Router_E1, Router_E2, Router_E3]
 
 #
-# Preparing docker network that will be used on Router.I2 and Router.E3 to
-# communicate with the broker instances
+# Addressing issues with python3 from RHSCL and selinux (pip)
 #
-- hosts: Router.I2,Router.E3
+- hosts: docker-host,Router.I2,Router.E3
   tasks:
+    # PYTHON3SCL - When using python3 from RHSCL the selinux module is failing
+    #              as it expects an /usr/bin/python3 executable to exist
+    #              A PR has been submitted to the selinux pip module to fix
+    #              this issue.
+    - name: Creating link for python ansible_python_interpreter in /usr/bin
+      command: "ln -s {{ ansible_python_interpreter }} /usr/bin/"
+      ignore_errors: yes
+    # END PYTHON3SCL
+    - name: Installing pip modules
+      pip:
+        executable: "{{ pip_binary }}"
+        name:
+          - docker
+          - selinux
+        extra_args: --user
```

Comment:
```
import_tasks: task/selinux_workaround.yml
```
or somethink like that? to use it here and up

Reply:
@nicob87 and @enkeys this has been applied and tested on both rhel7 and fedora 30.